index.src.html

```diff
@@ -1461,25 +1461,23 @@ <h3 id="generate-test-report-command">Generate Test Report</h3>
   2. Let |message| be the result of <a>trying</a> to get |parameters|'s
      {{GenerateTestReportParameters/message}} property.
 
-  3. If |message| is null, return a <a>WebDriver error</a> with <a>WebDriver
-     error code</a> <a>invalid argument</a>.
+  3. If |message| is not present, return a <a>WebDriver error</a>
+     with <a>WebDriver error code</a> <a>invalid argument</a>.
 
   4. Let |group| be the result of <a>trying</a> to get |parameters|'s
```

Comment:
Do we still need "trying" now that there's a default value?  Can we change it to "Let group be the value of parameters's [etc]"